lib/charliecloud.py

```diff
@@ -984,6 +984,15 @@ def manifest_for_download(self, image_ref):
    def unpack(self, image_ref):
       return self.unpack_base // image_ref.for_path
 
+   def image_unpacked_p(self, image_ref):
```

Comment:
1. This should take a path, not a reference. It answers the question "is this directory an image?".
2. Give that, it becomes not a method but rather a simple function, so put it in alphabetical order in the functions section.
3. I wonder if it should be called something else? Possibly `unpacked_image_p` but give it some thought.